spec/bundle/dsl_spec.rb

```diff
@@ -3,6 +3,18 @@
 require "spec_helper"
 
 describe Bundle::Dsl do
+  describe described_class::Entry do
+    subject(:entry) { described_class.new(:brew, entry_name) }
+
+    let(:entry_name) { :foo }
+
+    describe "#to_s" do
```

Comment:
Unless the coverage check fails without this: please remove this test because it's effectively just duplicating the (simple) implementation and the bug fixed is covered by the integration test above. Thanks!

Reply:
Hi @MikeMcQuaid - I will obviously remove it if that's what you want - it's your project! 😄But are you sure?

I know this is a trivial example, but I would argue it's not duplicating the implementation - the implementation is admittedly very simple, but it's a regression spec for other code that depends on that specific behaviour. The other spec might still pass due to unrelated changes.

Also, there are currently no specs for the `Entry` class, so the majority of that diff is just the boilerplate of adding the spec. It reasonable that it might have additional behaviour at some point?

Let me know.

Reply:
> But are you sure?

Yes, thanks.